core/model/uiuc/buildingdata.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2022 Board of Trustees of the University of Illinois.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package uiuc
```

Comment:
In general we want to put all files in a given package in the same directory. In this case, the `core/model` directory should only contain our internal models. These files in the `core/model/uiuc` directory should ideally be moved into their respective `driven/` package directories and placed in a file called `model.go`. This will also help us ensure that the core models remain generic so we can reuse the interface for other universities in the future.

Reply:
Will address when bringing in the identity building block